radixdlt/babylon-wallet-ios
# Remove most placeholder strings

## Description
- Removes all placeholder strings for missing resource names, instead there will be no text at all
- Follows the rule "use .symbol if present, else .name" for resources
- Uses .name for validators and dApps
- Removes the rule where dApps associated to a dApp must have a name in order to be shown

## How to test
Go through all places in the app where resources, validators and dApps appear, and try to find entities with .name and/or .symbol missing - make sure it looks reasonable and follows the rules

## PR submission checklist
